README.md

```diff
@@ -456,6 +456,11 @@ Params:
     readability. Use a non-capturing group (like `(?:abc)|(?:abc)`) if you need
     grouping without capturing.
 
+    Note that by default, RE2 doesn't use multiline mode, so ^ and $ will match
+    the start and end of the entire file, rather than each line. To enter
+    multiline mode you need to set the flag by including this: `(?m:YOUR_REGEX_HERE)`.
+    More information available in RE2 docs.
```

Comment:
nit: add a link to the RE2 doc?

Reply:
There was one in the previous paragraph, do you think another is useful?

Reply:
Don't bother then :)